src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,93 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final float INCREASE_CAPACITY = 2f;
+    private int loadFactor = (int) (DEFAULT_CAPACITY * DEFAULT_LOAD_FACTOR);
+    private int capacity = DEFAULT_CAPACITY;
```

Comment:
it's more hustle to keep capacity updated than the benefit of actually using it, delete the field, `table.length` is trivial enough